fe/fe-core/src/main/java/com/starrocks/catalog/DefaultExpr.java

```diff
@@ -1,9 +1,19 @@
 // This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
 package com.starrocks.catalog;
 
+import com.clearspring.analytics.util.Lists;
+import com.google.common.collect.ImmutableSet;
 import com.google.gson.annotations.SerializedName;
+import com.starrocks.analysis.Expr;
+import com.starrocks.analysis.FunctionCallExpr;
+import com.starrocks.analysis.FunctionName;
+
+import java.util.Set;
 
 public class DefaultExpr {
+
+    public static final Set<String> SUPPORTED_DEFAULT_FN = ImmutableSet.of("uuid()", "uuid_numeric()");
```

Comment:
Add `now()`?

Reply:
now is not the same logic. Currently, DEFAULT CURRENT_TIMESTAMP is currently supported, and the essence is now, but now should not be calculated by BE. So don't put it in yet.